core/emu.sh

```diff
@@ -1,35 +1,10 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
Do we actually need this?

Reply:
not really but kind of?

there's two reasons:
a) keep compatibility with people / scripts who type `emu.sh` -- but that could be solved with a symlink
b) it seems useful to have a _dependency-less_ wrapper, that tells you straight up what command you need

Reply:
I disagree with a), because people / scripts will benefit from the fact that there is `emu.sh` missing and look into the beautiful `emu.py` and all its features. b) seems reasonable. 

Again, no strong preference.

Reply:
i'm keeping it for now